.drone.yml

```diff
@@ -892,107 +892,6 @@ volumes:
 - name: dockersock
   temp: {}
 
----
-kind: pipeline
-type: kubernetes
-name: push-build-linux-arm-fips
```

Comment:
why did this disappear?

Reply:
I took it out deliberately - we don't build FIPS packages for 32-bit architectures on tag (like i386 or ARMv7) so it's pointless doing builds on push.